### PR TITLE
Finish import rewrites + deps

### PR DESCRIPTION
**Pull request by mkrizek**

### CODE REVIEW

File: migrate.py [hunk 1]
```diff
@@ -188,17 +188,17 @@ def visit_Assign(self, node):
     return plugin_data, deps
 
 
-def rewrite_imports(mod_src_text, coll, spec, namespace):
+def rewrite_imports(mod_src_text, collection, spec, namespace):
     """Rewrite imports map."""
-    plugins_path = ('ansible_collections', namespace, coll, 'plugins')
```
Comment (webknjaz) [inline migrate.py:193]: Why are you removing `ansible_collections`?

File: migrate.py [hunk 2]
```diff
@@ -213,32 +213,39 @@ def match_import_src(imp_src, import_map):
...
+            continue
+
+        if plugin_collection != collection:
+            deps.append(plugin_collection)
+            # FIXME I am changing a tuple!
+            exchange_new = list(exchange)
+            exchange_new[1] = plugin_collection
+            exchange = tuple(exchange_new)
+
         imp_src[:token_length] = exchange  # replace the import
```
Comment (webknjaz) [inline migrate.py]: It'd be cleaner this way:
```suggestion
         imp_src[:token_length] = exchange  # replace the import
         if plugin_collection != collection:
             imp_src[1:1] = [plugin_collection]
             deps.append(plugin_collection)
```

File: migrate.py [hunk 3]
```diff
@@ -213,32 +213,39 @@ def match_import_src(imp_src, import_map):
...
-    return mod_fst
+    return mod_fst, deps
```
Comment (webknjaz) [inline migrate.py]: Don't really need to return FST, it's modified in-place anyway:
```suggestion
    return deps
```

Comment (mkrizek) [reply migrate.py]: :+1:

File: migrate.py (hunk 2 above)
Comment (mkrizek) [reply migrate.py]: :+1:

File: migrate.py (hunk 1 above)
Comment (mkrizek) [reply migrate.py:193]: Sorry that was an oversight.

**Review by webknjaz — APPROVED** [review]
looks working

**Pull request merged**